bagger/README.md

```diff
@@ -0,0 +1,154 @@
+`rebach_bagger/config` directory. A config file can also be specified at
+runtime using `--config` or `-c`. Paths in the config file are all relative to
+the directory from which the script is called. Using absolute paths may be
+advisable if ReBACH-Bagger will be imported as a module.
+
+### Defaults
+
+The `Defaults` section of the configuration file contains keys that define the
+program's execution environment. The keys defined here can also be set at
+runtime using the command line options described above. Command line
+configuration options are first in precendce, meaning values passed with the
+command line will override values set in the config file.
+
+```toml
+[Defaults]
+output_dir = "out" # DART will output generated bags here; must be writable
+workflow = "default_workflow.json" # Path to the DART workflow file
+dart_command = "dart-runner" # Command or path to DART executable
+```
+
+### Logging
+
+ReBACH-Bagger logs errors, debug messages, and DART output to disk.
+
+```toml
+[Logging]
+log_dir = "logs" # Logger will write logs to this directory
+logfile_prefix = "ReBACH-Bagger" # Log filename prefix
+```
+
+### Wasabi
+
+Both DART and ReBACH-Bagger use the `access_key` and `secret_key` credentials
+defined in the configuration to authenticate to Wasabi. However, only
+ReBACH-Bagger uses the other variables defined here to access the storage
+endpoint for the purpose of checking for duplicate bags. DART uses the storage
+configuration embedded in the workflow JSON file for selecting the correct
+endpoint. Verify that these values match.
+
+```toml
+[Wasabi]
+name = "***override***"
+host = "***override***"
+bucket = "***override***"
+host_bucket = "***override***"
+access_key = "***override***"
+secret_key = "***override***"
+```
+
+### Metadata
+
+ReBACH-Bagger can apply a configurable set of metadata tags to the bags it
+generates. These tags are defined in the `Metadata` section of the
+configuration file. The tags are defined with the following schema:
+
+`tag_file.tag_name = "tag_source"`
+
+The `tag_source` element is a dot-notation path to the tag's corresponding key
+in the package's metadata JSON file. Take the following abbreviated example of
+Figshare's API metadata:
+
+```json
+{
+  "authors": [
+    {
+      "full_name": "Brian Avants"
+    },
+    {
+      "full_name": "Arno Klein"
+    }
+  ],
+  "license": {
+    "name": "Apache 2.0",
+    "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+  },
+  "title": "MNI space average T1-weighted MRI of the brain with anatomical labels.",
+  "doi": "10.6084/m9.figshare.2066037.v17"
+}
+```
+
+To define a set of tags named "First-Author", "License", and "DOI" in the "
+bag-info.txt" tag file, users can define the following relationships in the
+config file:
+
+```toml
+[Metadata]
+bag-info.First-Author = "authors.0.full_name"
+bag-info.License = "license.name"
+bag-info.DOI = "doi"
+```
+
+## DART Workflow
+
+## BagIt Profile
```

Comment:
Still a WIP? Would be useful to describe the process for obtaining DART and where it is recommended to be placed, and a short description to how to create/modify the workflows with a link to the DART docs. Also the difference between default_workflow.json and redata-wasabi.json

Reply:
I will complete these sections this week. At the moment, default_workflow.json and redata-wasabi.json are identical. I left the default file so named I thought it was a little clearer when included in the example config file.